### DISCUSSION TITLE
[feat] Tauri mobile Google Oauth 2.0

### DISCUSSION THREAD

**Discussion by NorbertBodziony** [tauri-apps/tauri#7955] [Q&A]
### Describe the problem

Hey i am looking for example or help of integrating google oauth with tauri mobile.

Thanks

### Describe the solution you'd like

I would prefer to open modal with safari inside that will close after user finished authorization.

### Alternatives considered

Currently, when i redirect user to google i am getting `disallowed_useragent` error. (I think we can't change user agent)

**Comment by Stephan-MC** [comment]
Please did 

> ### Describe the problem
> Hey i am looking for example or help of integrating google oauth with tauri mobile.
> 
> Thanks
> 
> ### Describe the solution you'd like
> I would prefer to open modal with safari inside that will close after user finished authorization.
> 
> ### Alternatives considered
> Currently, when i redirect user to google i am getting `disallowed_useragent` error. (I think we can't change user agent)
> 
> ### Additional context
> _No response_

Please did you find a workaround for this issue ? 🙏🏿

**Reply by WaterKnight1998** [reply]
I think the easiest way is using a deeplink as redirect uri. Did you find a solution??

**Reply by Stephan-MC** [reply]
still searching for a work around.
